wrap-template.js

```diff
@@ -0,0 +1,4 @@
+(function(){
```

Comment:
You should add parameters to your anonymous function:

    (function(window, document, undefined) {
        // ....
    })(window, document, void 0);